### DISCUSSION TITLE
Backdrop image overlay color

### DISCUSSION THREAD

**Discussion by thornbill** [jellyfin/jellyfin-androidtv#982] [General]
I find the current overlay color used for backdrop images makes text _very_ hard to read. We currently only use 55% opacity pure black for the overlay. In comparison web uses 86% opacity and the normal (almost but not quite black) background color.

I believe it would be best from a design stand point for us to use the same background color that we use normally instead of pure black. Although I am not a designer so I could be wrong on that. :joy: 

I do think we should be at least at 80% opacity though to improve readability based on the comparisons I did below.

**55% Opacity Black (current)**
![background 55 black (current)](https://user-images.githubusercontent.com/3450688/123305069-b097ad00-d4ed-11eb-976c-69956f8e0ae3.png)

**55% Opacity**
![background 55](https://user-images.githubusercontent.com/3450688/123305225-df158800-d4ed-11eb-958d-aa0a8132cfff.png)

**60% Opacity**
![background 60](https://user-images.githubusercontent.com/3450688/123305260-ea68b380-d4ed-11eb-8548-967e0bc3ddd9.png)

**70% Opacity**
![background 70](https://user-images.githubusercontent.com/3450688/123305308-f5234880-d4ed-11eb-88dc-f4cf657f173b.png)

**80% Opacity**
![background 80](https://user-images.githubusercontent.com/3450688/123305341-fe141a00-d4ed-11eb-9d3f-e154598e5b51.png)

**86% Opacity (used in web)**
![background 86](https://user-images.githubusercontent.com/3450688/123305432-17b56180-d4ee-11eb-9de1-7a1a1ca8a279.png)

**Comment by nielsvanvelzen** [comment]
70% opacity is the first one that is readable for me

**Comment by MrChip53** [comment]
I think maintaining consistency with web would be desirable (go with 86%).

**Reply by MrChip53** [reply]
Iirc the original ATV design had the text itself have another background behind just it so that is probably why we have a lighter overall opacity

**Reply by nielsvanvelzen** [reply]
Yes the first "row" (pretty much fills the complete screen) had a separate background but it looked really weird and the animation started to bug out at some point so I removed it.

**Comment by thornbill** [comment]
I opened a PR (https://github.com/jellyfin/jellyfin-androidtv/pull/983) changing it to 80% as a middle ground. We can adjust it further if needed.

**Reply by nielsvanvelzen** [reply]
Looks good, I'll merge this one and we can always change it again in another PR if necessary.
